docs/developer-docs/latest/setup-deployment-guides/deployment.md

```diff
@@ -6,21 +6,23 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guid
 
 # Deployment
 
-Strapi gives you many possible deployment options for your project or application. Strapi can be deployed on traditional hosting servers or services such as 21YunBox, Render, Heroku, AWS, Azure and others. The following documentation covers how to develop locally with Strapi and deploy Strapi with various hosting options.
+Strapi provides many deployment options for your project or application. Your Strapi applications can be deployed on traditional hosting servers or your preferred hosting provider. 
 
-::: strapi Database deployment
-Deploying databases along with Strapi is covered in the [databases guide](/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md#databases-installation-guides).
+The following documentation covers how to develop locally with Strapi and deploy Strapi with several common hosting options.
+
+::: strapi Community Guides
+In addition to the official deployment guides maintained by Strapi that are found here, community maintained guides for additional providers are available in the [Strapi Community Guides](https://forum.strapi.io/c/community-guides/28).
 :::
 
 ## General guidelines
 
 ### Hardware and software requirements
 
-To provide the best possible environment for Strapi there are a few requirements, these apply in both a development (local) as well as a staging and production workflow.
+To provide the best possible environment for Strapi the following requirements apply to development (local) and staging and production workflows.
 
-- Node LTS (v14 or v16) **Note that odd-number releases of Node will never be supported (e.g. v13, v15).**
-- NPM v6 or whatever ships with the LTS Node versions
-- Typical standard build tools for your OS (the `build-essentials` package on most Debian-based systems)
+- Node LTS (v14 or v16) **Odd-number releases of Node are not supported (e.g. v13, v15).**
```

Comment:
This is changing to node18. Alex has a separate PR. Just want to make sure there is no conflict with the merge.